src/rules/property-blacklist/README.md

```diff
@@ -0,0 +1,55 @@
+# property-blacklist
+
+Specify a blacklist of disallowed properties.
+
+```css
+    a { 
+      color: pink;
+    }
+/**    ↑ 
+ * These properties */
+```
+
+## Options
+
+`array`: `"["array", "of", "unprefixed", "properties"]`
+
+### `["array", "of", "unprefixed", properties"]`
+
+Blacklisted properties *must never* be used.
+
+Given:
+
+```js
+["background-size", "transform"]
```

Comment:
You're right, I should swap in a more realistic example. Off hand I can think of two, albeit edge-case, examples:

1. Your team makes a decision to do all animation and transitions in JS, so you want to blacklist all the `animation-*` and `transition-*` properties and enforce the decision with tooling for new devs on the project.
2. You want to avoid possibly-problematic properties like `text-rendering` and always rely on the browsers default of `text-rendering: auto` ([ref](http://bocoup.com/weblog/text-rendering/)).

Perhaps I'm clutching at straws here :) 

I think the `property-unit-black/whitelist` rules will be a lot more useful. I can think of lots of use cases for those like e.g. disallowing `px`s for `font-size` or only allowing `pt` for `line-height` etc.

Worth keeping this rule around to compliment the `property-unit-black/whitelist` one?